typed_ast/__init__.py

```diff
@@ -1 +1 @@
-
+__version__ = "1.1.0"
```

Comment:
The tradition we use in mypy is that immediately after tagging a release we add a new commit that bumps the version to <next-version>-dev. So here you would first submit and merge a diff that bumps the version to 1.1.1-dev, and once that's landed you'd merge this one (but with 1.1.1-dev in there somewhere).